src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataMaskingTests.cs

```diff
@@ -25,12 +25,13 @@ public class DataMaskingTests : SqlTestsBase
     {
         protected override void SetupManagementClients(RestTestFramework.MockContext context)
         {
-            var sqlCSMClient = GetSqlClient();
+            var sqlClient = GetSqlClient(context);
```

Comment:
@jaredmoo that's all I wanted to know that you are aware of it, that is the whole point of code review :). My intention is to understand if this was overlooked or was intentional.
Regarding going back and forth on refactoring, please understand this is a huge PR, GitHub is challenging when it comes to large PRs.